### PR TITLE
Added blastoff.c, blastoff.go, and todo.txt

### PR DESCRIPTION
**Pull request by spurgeonc**
Does this look good, everyone?

### CODE REVIEW

**Comment by gleeg2** [comment]
Code looks good to me.

**Review by ghost — APPROVED** [review]
Code looks good to me. 👍
I think the todo.txt isn't needed since you have the asciinema link.

**Comment by nay7** [comment]
Looks good nice job!

**Review by nay7 — APPROVED** [review]
nice job

File: blastoff.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package main
+
+import "fmt"
+
+func main() {
+	const Time int = 10
```
Comment (defreez) [inline blastoff.go:6]: Start this variable name with a lower case. It doesn't make any difference here, but capital first letters in Go have meaning.

File: blastoff.c [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
+#include <stdlib.h>
+
+int main(void) {
+  const int TIME_TO_LAUNCH = 10;
+  for (int i = TIME_TO_LAUNCH; i >=0; i--){
```
Comment (defreez) [inline blastoff.c:6]: Use clang-format

**Review by defreez — COMMENTED** [review]
LGTM

**Pull request merged**